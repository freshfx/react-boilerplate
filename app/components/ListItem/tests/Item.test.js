import 'jest-styled-components'
import React from 'react'
import {mount} from 'enzyme'

import Item from '../Item'

const renderComponent = (props = {}) => mount(<Item {...props} />)

describe('<Item />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderComponent()
    expect(renderedComponent).toMatchSnapshot()
  })
})

