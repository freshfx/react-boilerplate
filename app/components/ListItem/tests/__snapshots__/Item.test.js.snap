// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
  height: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<Item>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "Item-sc-3y9mie-0",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            "display:flex;justify-content:space-between;width:100%;height:100%;align-items:center;",
          ],
        },
        "displayName": "Item",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "Item-sc-3y9mie-0",
        "target": "div",
        "toString": [Function],
        "usesTheme": false,
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <div
      className="c0"
    />
  </StyledComponent>
</Item>
`;
